StyleCop.Analyzers/StyleCop.Analyzers/SpacingRules/SA1026CodeFixProvider.cs

```diff
@@ -42,22 +42,22 @@ public override FixAllProvider GetFixAllProvider()
                     continue;
                 }
 
-                SyntaxToken token = root.FindToken(diagnostic.Location.SourceSpan.Start);
-                if (token.IsMissing)
-                {
-                    continue;
-                }
-
-                context.RegisterCodeFix(CodeAction.Create(SpacingResources.SA1026CodeFix, t => GetTransformedDocumentAsync(context.Document, root, token)), diagnostic);
+                context.RegisterCodeFix(CodeAction.Create(SpacingResources.SA1026CodeFix,
+                                        token => GetTransformedDocumentAsync(context.Document, diagnostic.Location.SourceSpan.Start, token)), 
```

Comment:
:bulb: Avoid aligning arguments like this. If you want to align arguments, move the first argument to the next line as well, like this:

```csharp
context.RegisterCodeFix(CodeAction.Create(
    SpacingResources.SA1026CodeFix,
    token => ...,
    ...);
```